.travis.yml

```diff
@@ -32,10 +32,12 @@ before_script:
   fi
 jobs:
   include:
-    - stage: test
-      script:
-        - npm install || exit 1
-        - npm run test || exit 1
+    # There are no JS blocks to test yet. Uncomment below when we start adding
```

Comment:
Are you sure this wouldn't work if left uncommented? Now that only the JS tests are going to run, the `exit 1` should work.

Reply:
When I run it locally, `npm run test-js` returns an error code because there are no tests. The `exit 1` is also a failure code. If we changed it to `exit 0` then it would succeed, but then there would be very little point to having it in CI at all (if a test fails, the CI would still "pass").